lookout/style/format/feature_extractor.py

```diff
@@ -360,6 +361,11 @@ def composite_class_printables(self) -> List[str]:
         return ["".join(CLASS_PRINTABLES[label] for label in labels)
                 for labels in self.labels_to_class_sequences]
 
+    def label_to_str(self, label: int) -> str:
+        """Convert a label to string."""
+        return "".join(INDEX_CLS_TO_STR[cls]
+                       for cls in self.labels_to_class_sequences[label])
```

Comment:
Mmm, I did not mention it earlier, but we have this guy in `FeatureExtractor`: 
https://github.com/src-d/style-analyzer/blob/a38de6d64803ae85ed835411d327fb27e954e123/lookout/style/format/feature_extractor.py#L344

Reply:
ok, it is different.